fix.sh

```diff
@@ -152,6 +152,17 @@ xmlstarlet ed -L \
 	-u '//_:function[@name="x11_lookup_xdisplay"]//_:type[@name="xlib.Display"]/@c:type' -v "gpointer" \
 	GdkX11-4.0.gir
 
+xmlstarlet ed -L \
+	-u '//_:callback[@name="Win32MessageFilterFunc"]//_:type[@name="win32.MSG"]/@c:type' -v "gpointer" \
+	-u '//_:class[@name="Win32HCursor"]//_:type[@name="win32.HCURSOR"]/@c:type' -v "gssize" \
+	-u '//_:class[@name="Win32Surface"]//_:type[@name="win32.HGDIOBJ"]/@c:type' -v "gssize" \
+	-u '//_:class[@name="Win32Surface"]//_:type[@name="win32.HWND"]/@c:type' -v "gssize" \
+    -u '//_:function[@name="win32_handle_table_lookup"]//_:type[@name="win32.HWND"]/@c:type' -v 'gssize' \
+    -u '//_:function[@name="win32_set_modal_dialog_libgtk_only"]//_:type[@name="win32.HWND"]/@c:type' -v 'gssize' \
+    -u '//_:function[@name="win32_icon_to_pixbuf_libgtk_only"]//_:type[@name="win32.HICON"]/@c:type' -v 'gssize' \
+    -u '//_:function[@name="win32_pixbuf_to_hicon_libgtk_only"]//_:type[@name="win32.HICON"]/@c:type' -v 'gssize' \
```

Comment:
```suggestion
	-u '//_:callback[@name="Win32MessageFilterFunc"]//_:type[@name="Win32MSG"]/@c:type' -v "gpointer" \
	-u '//_:class[@name="Win32HCursor"]//_:type[@name="Win32HCURSOR"]/@c:type' -v "gssize" \
	-u '//_:class[@name="Win32Surface"]//_:type[@name="Win32HGDIOBJ"]/@c:type' -v "gssize" \
	-u '//_:class[@name="Win32Surface"]//_:type[@name="Win32HWND"]/@c:type' -v "gssize" \
    -u '//_:function[@name="win32_handle_table_lookup"]//_:type[@name="Win32HWND"]/@c:type' -v 'gssize' \
    -u '//_:function[@name="win32_set_modal_dialog_libgtk_only"]//_:type[@name="Win32HWND"]/@c:type' -v 'gssize' \
    -u '//_:function[@name="win32_icon_to_pixbuf_libgtk_only"]//_:type[@name="Win32HICON"]/@c:type' -v 'gssize' \
    -u '//_:function[@name="win32_pixbuf_to_hicon_libgtk_only"]//_:type[@name="Win32HICON"]/@c:type' -v 'gssize' \
```

Otherwise it would expect a separate library dependency, which is causing the compile / generation error you have.

Reply:
Actually never mind. I just noticed you were making use of the win32 gir file